app/news/list/news-page.module.css

```
@@ -0,0 +1,75 @@
+.header {
+    justify-content: space-between;
+    align-items: center!important;
+    padding:20px;
+    background-color:#0b69eb;
+    background: linear-gradient(90deg, #0b69eb 60%, #ffffff 100%);
```

Stylelint (17.12.0)

[error] Overridden property "background-color" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)